stringslab.js

```diff
@@ -0,0 +1,158 @@
+// # String Exercises
+//
+// 1. `console.log` the first character of a string.
+let quote = 'A friend is one that knows you as you are, \nunderstands where you have been, \naccepts what you have become, \nand still, gently allows you to grow'
+
+console.log(quote[0]);
+
+// 2. `console.log` the length of a string.
+console.log(quote.length);
+//
+// 3. `console.log` the last character of any string.
+console.log(quote[quote.length - 1]);
+
+
+// 4. Create a code block that takes a single string variable,
+// and returns a copy of the string with the last letter
+// capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+
+let newQuote = quote.slice(0, quote.length - 1) + quote[quote.length - 1].toUpperCase()
+console.log(newQuote)
+
+
+// 5. Create a drEvil code block that will take a single
+//number variable, and log the '<variableAmount> dollars',
+// your code block should add '(pinky)' at the end of the
+ //amount if it's 1 million. For example:
+//
+
+let x = 10
```

Comment:
Not exactly what the question as asking here. Solving this problem would involve if statements. Try to solve this again.